Makefile

```diff
@@ -221,12 +221,13 @@ test-skip-slow:
 fast-test-skip-slow:
 	@stack test --fast ${STACKFLAGS} ${STACKTESTFLAGS} --ta '-p "! /slow tests/"'
 
-SHELLTEST := $(shell command -v shelltest 2> /dev/null)
+SMOKE := $(shell command -v smoke 2> /dev/null)
 
-.PHONY : test-shell
-test-shell : install
-	@$(if $(SHELLTEST),, stack install shelltestrunner)
-	shelltest ${SHELLTESTFLAGS} tests
+.PHONY : smoke
+smoke: install
+	@$(if $(SMOKE),, $(error "Smoke not found, please install from the sources"))
```

Comment:
Perhaps add a link to the smoke project page to help the user find out how to install it? (especially as it's not easy to search for this project by name).